math/inv_of_polynomials/info.toml

```
@@ -0,0 +1,15 @@
+title = 'Inv of Polynomials'
```

```suggestion
title = 'Inv of Polynomial'
```

ついでにフォルダ名もinv_of_polynomials -> inv_of_polynomial

---

いやこれは勘違いでした